packages/appstore/src/stores/config-store.ts

```diff
@@ -1,14 +1,3 @@
-import { TConfigProps } from 'Types';
 import BaseStore from './base-store';
 
-export default class ConfigStore extends BaseStore {
```

Comment:
why these changes are here?

Reply:
Not very sure about this changes, @yashimwong did this for implementing router configuration to app store.

Reply:
lets keep it as it it was for now, will change it later on if needed